src/Test.hs

```diff
@@ -29,14 +29,13 @@ main = putStrLn $ tableString [ column (expandUntil 30) left (charAlign ':') def
     bigNum    = "200300400500600.2"
     smallNum  = "4.20000000"
     styles    = [ asciiS
-                , asciiDoubleS
                 , asciiRoundS
                 , unicodeS
                 , withoutBorders unicodeS
                 , unicodeRoundS
                 , unicodeBoldS
                 , unicodeBoldStripedS
-                , unicodeBoldHeaderS
+                -- , unicodeBoldHeaderS
```

Comment:
What's happening here?

Reply:
I'm commenting out `unicodeBoldHeader` because the functionality to render this style is not yet introduced at this point in the PR. It becomes available later, and so is re-introduced then.

Reply:
Can you not move it before this commit (i.e. with `rebase`)?

Reply:
I could, but I hesitate to do so because this would make a giant commit with lots of different functionality introduced at once, which makes it difficult to see what each individual thing does. This may reflect a difference in approach to commits between the two of us.

Reply:
I don't know.  What I do not like is intermediate states between commits.  My current view on the matter is the following.  Commits should be able to exist on their own.  The functionality should be atomic in a sense.  What I also don't like is monolithic commits where I as a reviewer cannot identify what feature is implemented.

As far as I can tell, the Unicode stuff is independent.  If you think it's completely impossible then feel free to resolve it.  But then I would appreciate a comment in the code why this is there.